src/modules/deposit/controller.ts

```diff
@@ -15,6 +15,14 @@ export class DepositController {
     return this.depositService.getDeposits(query.status, limit, offset);
   }
 
+  @Get("deposits/:address")
```

Comment:
What do you think if we make the `address` a query parameter as well? This means that we can re-use the endpoint from above. Usually endpoint parameters are used to identify resources (by an unique id for example). IN our case the address is more like a filtering field

Reply:
Yea that is a good point. Will change this

Reply:
Changed it [here](https://github.com/across-protocol/scraper-api/pull/182/commits/0dade652216858a0130582b1c0cdd3975942c3a3)